lib/shopify_dashboard_plus.rb

```diff
@@ -49,39 +73,47 @@ def set_connection(key, pwd, name)
       ShopifyAPI::Base.site = shop_url
       shop = ShopifyAPI::Shop.current
       $shop_name = name
-      connection_open
+      open_connection
     rescue
-      connection_closed
+      close_connection
     end
   end
 
-  def connection_closed
+  def close_connection
     $connected = false
   end
 
-  def connection_open
+  def open_connection
     $connected = true
   end
 
-  def connected?
-    $connected
-  end
+  def connected?; $connected; end
 
-  def shop_name
-    $shop_name
-  end
+  def shop_name; $shop_name; end
 
 
   ## Generic Helpers
+  def date_today
+    DateTime.now.strftime('%Y-%m-%d')
+  end
+
+  def days(num)
+    num.to_i == 1 ? "#{num} Day" : "#{num} Days"
+  end
+  
   def hash_to_list(unprocessed_hash)
     return_list = []
-    unprocessed_hash.each do |key, value|
-      return_list.append([key, value])
-    end
+    unprocessed_hash.each {|key, value| return_list.append([key, value])}
```

Comment:
This might be cleaner using [`Enumerable#map`](http://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-map)

```ruby
def hash_to_list(unprocessed_hash)
  unprocessed_hash.map { |key, value| [kev, value] }
end
```

but [`Hash#to_a`](http://ruby-doc.org/core-2.2.2/Hash.html#method-i-to_a) will do this for you

```
irb(main):001:0> {foo: "bar"}.to_a
=> [[:foo, "bar"]]
```